kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -0,0 +1,53 @@
+package com.kodilla.testing.collection;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class CollectionTestSuite {
+
+    @Before
+    public void before() {
+        System.out.println("Test Case: start");
+    }
+
+    @After
+    public void after() {
+        System.out.println("Test Case: end");
+    }
+
+    @Test
+    public void testOddNumbersExterminatorEmptyList() {
+        OddNumbersExterminator testExt1 = new OddNumbersExterminator();
+        ArrayList<Integer> numbers = new ArrayList<Integer>();
+        ArrayList<Integer> testCaseEmptyList = testExt1.exterminate(numbers);
+        if(testCaseEmptyList.size() == 0) {
+            System.out.println("Test \"testOddNumbersExterminatorEmptyList\" ok");
+        } else {
+            System.out.println("Test \"testOddNumbersExterminatorEmptyList\" failed!");
+        }
```

Comment:
Trzymaj sie prosze formuly //Given //When //Then

Reply:
system out'y sa zbedne w testach. Brakuje za to assercji. Np
`Assert.assertEquals(0, testCaseEmptyList.size());`